watches/commands/base.py

```diff
@@ -46,6 +51,142 @@ def getResponseContentType(self):
         """Response MIME type. By default we assume JSON, make sure to override if needed."""
         return self.JSON_APPLICATION
 
+    def transformData(self, data):
+        """
+        Data can be transformed before sending to client.
+        Currently, the only transformation type implemented is 'nested'.
+        :param data:
+        :return:
+        """
+        transform = self.options['--transform']
+        if transform and transform is not None and transform == "nested":
```

Comment:
`if transform and transform is not None` - the part `transform is not None` is redundant - the `if transform` part already does that.
Should throw an exception or print an error or something if the value is specified and it is not equal to `"nested"`